### DISCUSSION TITLE
Add --branch flag to force:package:version:list

### DISCUSSION THREAD

**Discussion by goto-dev-null** [forcedotcom/cli#2382] [Ideas]
**Is your feature request related to a problem? Please describe.**
Feature enhancement. It would be really helpful to filter by branch when listing package versions.

**What are you trying to do**
Quickly view package versions on a specific branch.

**Describe the solution you'd like**
--branch will filter the results based on its "branch" attribute.

**Describe alternatives you've considered**
Powershell or grep. Which, mind you, is doable, but seems like it should be unnecessary.

[a comment by an automated account is not shown]

**Comment by uip-robot-zz** [comment]
This issue has been linked to a new work item: [W-11167396](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000xL6yIYAS/view)

[a comment by an automated account is not shown]
